salt/modules/disk.py

```diff
@@ -464,11 +479,18 @@ def fstype(device):
     if salt.utils.which('df'):
         # the fstype was not set on the block device, so inspect the filesystem
         # itself for its type
-        df_out = __salt__['cmd.run']('df -T {0}'.format(device)).splitlines()
-        if len(df_out) > 1:
-            fs_type = df_out[1]
-            if fs_type:
-                return fs_type
+        if __grains__['kernel'] == 'AIX' and os.stat('/usr/sysv/bin/df'):
```

Comment:
Are you trying to check for the existence of `/usr/sysv/bin/df` here? If so, `os.stat` is not what you want. If that file does not exist, you will get an `OSError`, not a result of `False`.